x11-packages/xorg-server/xorg-server-xephyr.subpackage.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_SUBPKG_DESCRIPTION="nested X server"
+TERMUX_SUBPKG_DEPEND_ON_PARENT="no"
+TERMUX_SUBPKG_DEPENDS="libandroid-shmem, libpixman, libxau, libxcb, libxdmcp, libxfont2, opengl, openssl, xorg-xkbcomp, xcb-util, xcb-util-image, xcb-util-keysyms, xcb-util-renderutil, xcb-util-wm"
```

Comment:
```suggestion
TERMUX_SUBPKG_DEPENDS="libandroid-shmem, libpixman, libxau, libxcb, libxdmcp, libxfont2, opengl, openssl, xcb-util, xcb-util-image, xcb-util-keysyms, xcb-util-renderutil, xcb-util-wm, xorg-xkbcomp, xkeyboard-config"
```

Reply:
I am done. You can squash after this.

Reply:
It has lot of commits, it will be better to squash.